packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1270,7 +1271,7 @@ function mountEffect(
     }
   }
   return mountEffectImpl(
-    UpdateEffect | PassiveEffect,
+    UpdateEffect | PassiveEffect | PassiveStaticEffect,
```

Comment:
Nit: Let's move this to `mountEffectImpl`. Makes it less likely we'll forget about it.

Reply:
`mountEffectImpl` is used for layout effects also. This static flag only applies to passive effects.